pkgs/applications/misc/mepo/default.nix

```diff
@@ -53,31 +69,98 @@ in stdenv.mkDerivation rec {
+      --suffix PATH : $bin/bin
+    wrapProgram $bin/bin/mepo_ui_menu_pin_editor.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox ]}
+    wrapProgram $bin/bin/mepo_ui_menu_pref_fontsize.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox ]}
+    wrapProgram $bin/bin/mepo_ui_menu_pref_url.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox findutils ]}
+    wrapProgram $bin/bin/mepo_ui_menu_pref_zoom.sh\
+      --suffix PATH : ${makeBinPath [ busybox ]}
+    wrapProgram $bin/bin/mepo_ui_menu_reposition_nominatim.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox curl jq ]}
+    wrapProgram $bin/bin/mepo_ui_menu_route_graphhopper.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox curl jq]}
+    wrapProgram $bin/bin/mepo_ui_menu_route_overpassrelation.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox curl findutils jq ]}
+    wrapProgram $bin/bin/mepo_ui_menu_search_nominatim.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox curl jq ]}
+    wrapProgram $bin/bin/mepo_ui_menu_search_overpass.sh\
+      --suffix PATH : $bin/bin:${makeBinPath [ busybox curl jq ]}
+      wrapProgram $bin/bin/mepo_ui_menu_user_pin_updater.sh\
+      --suffix PATH : ${makeBinPath ([ busybox curl jq ] ++ (optional withGpsd gpsd))}
+    wrapProgram $bin/bin/mepo --suffix PATH : $bin/bin
+  '' ]);
+
+  outputs = [ "bin" "out" "man" ];
 
   meta = with lib; {
     description = "Fast, simple, and hackable OSM map viewer";
+    longDescription = ''
+      # Menu Systems
+      Mepo supports multiple programs for displaying the menu. The default here is zenity.
+      If you prefer to use another, chose the `mepo-dmenu`, `mepo-bemenu` or `mepo-framebuffer`
+      packages, respectively.
+
+      # Location
+      For getting Location data, Mepo accepts 3 Backends:
+      - GPSD
+      - Geoclue
+      - Mozilla Location Services
+
+      Both GPSD and Geoclue require extra setup, that's why they're both set to false by default.
+
+      For GPSD, you need to enable (and configure) GPSD. This means your device must have a GPS sensor.
+      If you're on NixOS, see the
+      [gpsd configuration options](https://search.nixos.org/options?type=packages&query=services.gpsd)
+      NixOS provides.
+
+      To configure geoclue, you need to install the geoclue service and allow the `where-am-i`
+      program mepo uses. To do that, on NixOS:
+      ```nix
+      services.geoclue2 = {
+        enable = true;
+        appConfig.where-a-i = {
+          isAllowed = true;
+          isSystem = false;
+        };
+      };
+
+      ```
+      Finally, you need to override this package, setting `withGeoclue` or `withGpsd` to true, respectively.
+      (It is also possible to enable and configure both.)
+      To override the package, in your `environment.systemPackages`, list your mepo package of choice as
+      `(mepo.override {withGpsd = true; withGeoclue = true;}`. (only enable what you actually have set up).
```

Comment:
I suggest making this a bit more concise (I figure you wrote it while sleep-deprived):

```suggestion
      ```
      Finally, you need to override this package, setting `withGeoclue`
      and/or `withGpsd` to true, respectively, e.g. list your mepo package
      of choice as `(mepo.override { withGpsd = true; withGeoclue = true; })`.
```

Note that the overridden declaration also works with nix-shell.